BuildAddOn.py

```diff
@@ -200,25 +200,23 @@ def GetToolset (version):
     return 'v143'
 
 
-def GetProjectGenerationParams (workspaceRootFolder, buildPath, addOnName, platformName, devKitFolder, version, languageCode, additionalParams):
+def GetProjectGenerationParams (workspaceRootFolder, buildPath, platformName, devKitFolder, version, additionalParams):
     # Add params to configure cmake
     projGenParams = [
         'cmake',
         '-B', str (buildPath)
     ]
 
     if platformName == 'WIN':
-        vsGenerator = GetInstalledVisualStudioGenerator ()
-        projGenParams.append (f'-G {vsGenerator}')
-        toolset = GetToolset (int (version))
-        projGenParams.append (f'-T {toolset}')
+        projGenParams.extend ([
+            '-G', GetInstalledVisualStudioGenerator (),
+            '-T', GetToolset (int (version)),
+        ])
     elif platformName == 'MAC':
-        projGenParams.extend (['-G', 'Xcode'])
+        projGenParams.append ('-GXcode')
 
     projGenParams.append (f'-DAC_VERSION={version}')
-    projGenParams.append (f'-DAC_ADDON_NAME={addOnName}')
     projGenParams.append (f'-DAC_API_DEVKIT_DIR={str (devKitFolder / "Support")}')
-    projGenParams.append (f'-DAC_ADDON_LANGUAGE={languageCode}')

```

Comment:
These are now derived from the config.json file in CMake.

Reply:
Oops, not the language one actually, since this is needed to build all languages in bulk via this script.